src/main/resources/application.yml

```diff
@@ -23,3 +23,7 @@ jwt.public-key-path: ${JWT_PUBLIC_KEY_PATH:}
 crypt4gh:
   private-key-path: ${CRYPT4GH_PRIVATE_KEY_PATH:}
   private-key-password-path: ${CRYPT4GH_PRIVATE_KEY_PASSWORD_PATH:}
+
```

Comment:
Remove this, please. I'm in progress of adding this functionality - it will conflict.

Reply:
If it something else that is needed for the functionality it can wait or you can cherrypick the commits in the end.